src/rc.ts

```diff
@@ -33,13 +35,15 @@ export interface RC {
    * @default `npm`
    */
   strategy?: vuln.Strategy.Kind;
+  /** NodeSecure scanner Object configuration */
+  scanner?: ScannerConfiguration;
   /** NodeSecure ci Object configuration */
   ci?: CiConfiguration;
   /** NodeSecure report Object configuration */
   report?: ReportConfiguration;
 }
 
-export type RCGenerationMode = "minimal" | "ci" | "report" | "complete";
+export type RCGenerationMode = "minimal" | "ci" | "report" | "scanner" | "complete";
```

Comment:
Sync this change in the README ?
https://github.com/NodeSecure/rc/blob/49402d9a4cffd854d2f2f3ce1927fb543680c8f5/README.md?plain=1#L197